discord/__main__.py

```diff
@@ -152,44 +160,68 @@ async def cog_after_invoke(self, ctx):
         # called after a command is called here
         pass
 
-'''
+"""
 
 
 # certain file names and directory names are forbidden
 # see: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247%28v=vs.85%29.aspx
 # although some of this doesn't apply to Linux, we might as well be consistent
 _base_table = {
-    '<': '-',
-    '>': '-',
-    ':': '-',
-    '"': '-',
+    "<": "-",
+    ">": "-",
+    ":": "-",
+    '"': "-",
     # '/': '-', these are fine
     # '\\': '-',
-    '|': '-',
-    '?': '-',
-    '*': '-',
+    "|": "-",
+    "?": "-",
+    "*": "-",
 }
 
 # NUL (0) and 1-31 are disallowed
 _base_table.update((chr(i), None) for i in range(32))
 
 _translation_table = str.maketrans(_base_table)
 
+
 def to_path(parser, name, *, replace_spaces=False) -> Path:
     if isinstance(name, Path):
         return name
 
-    if sys.platform == 'win32':
-        forbidden = ('CON', 'PRN', 'AUX', 'NUL', 'COM1', 'COM2', 'COM3', 'COM4', 'COM5', 'COM6', 'COM7', \
-                     'COM8', 'COM9', 'LPT1', 'LPT2', 'LPT3', 'LPT4', 'LPT5', 'LPT6', 'LPT7', 'LPT8', 'LPT9')
+    if sys.platform == "win32":
```

Comment:
This seems like a step back to be honest (I know it's auto-formatted, but then we'd need to add ignore/skip everywhere)

Reply:
I disagree. These are both formatting changes only, and the current code violates the recommended line length limits. Reading the current code with less horizontal screen space is much more difficult before the changes.

No code logic has been changed at all with these two changes. It's simply been changed to adhere to the Black code style: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html